csm/core/services/hotfix_update.py

```diff
@@ -47,22 +47,23 @@ async def upload_package(self, file_ref, file_name):
 
         model = await self._get_renewed_model(const.SOFTWARE_UPDATE_ID)
         if model and model.is_in_progress():
+            Log.error("You cannot upload a new package while there is an ongoing update")
```

Comment:
Need to print specifics like model etc along with the error. Generic error should be avoided.

Reply:
Comment addressed/.